package/MDAnalysis/analysis/rms.py

```diff
@@ -739,6 +740,12 @@ class RMSF(AnalysisBase):
     in the array :attr:`RMSF.results.rmsf`.
 
     """
+
+    @classmethod
+    @property
+    def available_backends(cls):
```

Comment:
Rather than make the default "all the backends", can we please make the default "local" and and just have `available_backends` go through a validator of types?

Otherwise by default anything in the wild that inherits from AnalysisBase (both here and all over the downstream ecosystem) is suddenly going to find itself with a ton of available backends.

Reply:
Yep, I didn't think about the downstream projects. The MDAnalysis itself will then have a `available_backends` for each class explicitly, but I guess it's not a problem.

Reply:
Resolving as outdated.